enigma/src/main/java/cuchaz/enigma/analysis/EntryReference.java

```diff
@@ -1,144 +1,153 @@
-/*******************************************************************************
```

Comment:
Same as in #336, the diffs here are weirdly large.

Reply:
Turns out it's easily fixed with git. Please review.